service_notification/src/main/kotlin/org/thechance/service_notification/domain/usecases/IUserUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package org.thechance.service_notification.domain.usecases
+
+import org.thechance.service_notification.domain.model.User
+
+interface IUserUseCase {
+    suspend fun createUser(user: User): Boolean
```

Comment:
I believe Bareq said that interfaces should be in the same file as their implementation.